recipes/unitig-caller/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "unitig-caller" %}
-{% set version = "1.0.0" %}
+{% set version = "1.0.1" %}
 
```

Comment:
```suggestion
{% set sha256 = "451d74bbb5c32caef812d0897ae4e3745c6d3bfe02af9371c72d6299a4493f1f" %}
```

Reply:
I know everyone else does it, but IMO this just adds clutter to the recipe because it’s a variable that is used only once.

Reply:
I’m saying this only because you turned this into a "Suggested change".

If you’re comfortable, feel also free to make improvements to PRs by just editing them. It’s a bit more efficient that way. (And then I wouldn’t have mentioned the sha256 variable.)